src/helpers/stepFunctions.ts

```diff
@@ -57,7 +56,9 @@ export default class StepFunctions {
           exec.status === "RUNNING"
       ).length !== 0
     ) {
-      continue;
+      executionList = await this.stepFunctions
```

Comment:
You could add a small delay (between 50ms and 1s) in order to avoid completely spamming out the API